common-pipeline/datarecord-metadata/datarecord-metadata-v3-schema.json

```diff
@@ -149,7 +149,14 @@
           "minItems": 0,
           "items": {
             "description": "A value discovered in the data",
-            "type": "Can be a primitive type (string, integer, double, date, time, timestamp, binary, boolean)"
+            "type": [
```

Comment:
Let's remove object and array for now.  I don't think anyone actually supports those.

Reply:
I also don't think we need integer.  I think number covers integers and doubles.

Reply:
Removed object, array, integer types